packages/superset-ui-core/test/query/processExtraFormData.test.ts

```diff
@@ -70,6 +70,7 @@ describe('overrideExtraFormData', () => {
           time_range: '100 years ago',
         },
         {
+          // @ts-expect-error
```

Comment:
what this mean?

Reply:
This was added in TS 3.9: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments

Reply:
👍